README.md

```diff
@@ -39,9 +39,9 @@ For a list of files ff(1) picks up, visit
 
 There is a Makefile defaulting to clang. It will work with gcc aswell: `make CC=gcc`.
 
-It's usually best to use `build-clang.cmd` on Windows and `build-clang.sh` on
-Linux, as you can add additional arguments easily. Reminder that you should add
-`-O3` (or `/Ot` for cl).
+It's usually best to use `build-clang.cmd` on Windows, as you can add additional arguments easily. 
+Reminder that you should add `-O3` (or `/Ot` for cl).
+There is also `build-clang.sh` for Unix-like systems, should you wish not to use the Makefile.
```

Comment:
Why "should you wish not"?

Reply:
uhm theoretically make could be missing? I dunnoooooo